core/trino-spi/src/test/java/io/trino/spi/block/TestingBlockEncodingSerde.java

```diff
@@ -29,7 +29,11 @@
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static java.util.Objects.requireNonNull;
 
-// This class is exactly the same as BlockEncodingManager. They are in SPI and don't have access to InternalBlockEncodingSerde.
+/**
+ * This class is only meant to be used within the trino-spi module. For all other modules, use InternalBlockEncodingSerde.TESTING_BLOCK_ENCODING_SERDE
+ * <p>
+ * This class is exactly the same as BlockEncodingManager. They are in SPI and don't have access to InternalBlockEncodingSerde.
```

Comment:
"They"?

Reply:
That comes from the old comment that was there.